packages/istio/charts/istio-1.6.3/templates/file-mount.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
```

Comment:
Same, needs to be versioned and match pattern. And would be good to be more descriptive